uint8   balance                 # Remain of liquid
float32 ground_speed            #
uint8   pump                    # pump operation duty
